### DISCUSSION TITLE
Onvif test question

### DISCUSSION THREAD

**Discussion by Lindsey118** [onvif/testspecs#77] [Q&A]
[Test tool version]
	23.12

[Description]
We are failed to pass following two RTSP test items:
	RTSS-1-1-46-v21.12 VIDEO ENCODER CONFIGURATION – JPEG RESOLUTION
	RTSS-1-1-53-v21.12 MEDIA STREAMING – JPEG (VALIDATING RTP HEADER EXTENSION)

And we find out the possible cause would be the MTU size setting of the RTP packet.
1.	We wonder if there’s a limitation of this MTU size support of RTSP on the conformance test tool? 
2.	If “1” is yes, will the test tool support the larger MTU size?
3.	Or is there any suggestion?

**Comment by madhurao68** [comment]
Hi,
We will need Wireshark logs to help us better understand and provide you with our recommendation. Thanks.

**Comment by Lindsey118** [comment]
Hello,
Thank you for checking this issue.
Please see attached files for the wireshark packet and onvif test tool log.
Thank you.
[OnvifLog.zip](https://github.com/onvif/testspecs/files/15256826/OnvifLog.zip)

**Comment by Lindsey118** [comment]
Hello @madhurao68 ,
Can you provide your recommendation according to onvif wireshark to us?
Thank you.

**Comment by madhurao68** [comment]
Hi,
Could you please try test cases with a bigger value in Operation Delay? E.g. 5-10 minutes. Current value as per the logs is 5 s.
We are not sure if that will help but we want to eliminate this problem with frames during operation delay.
Thanks for your patience!

**Comment by Lindsey118** [comment]
Hello @madhurao68 ,
Thank you for the reply.
However, we try to change operation delay to 5 minutes and the behavior is the same. 
Attachments are test tool log about this experiment.
[TestToolLog.zip](https://github.com/user-attachments/files/15836951/TestToolLog.zip)
